spec/factories/physical_server.rb

```diff
@@ -1,4 +1,5 @@
 FactoryGirl.define do
   factory :physical_server do
+    vendor "Lenovo"
```

Comment:
Shouldn't this be lower case because of https://github.com/ManageIQ/manageiq/pull/15128/files#diff-43874eb28d9d57e685f11f1c9ec137c6R10 ?